src/main/java/com/williamfiset/algorithms/sorting/RadixSort.java

```diff
@@ -1,8 +1,44 @@
-/** NOTE: This file is still under development */
 package com.williamfiset.algorithms.sorting;
 
 public class RadixSort {
+
+  public static int getMax(int[] array) {
+    int max = array[0];
+    for (int i = 0; i < array.length; i++) {
+      if (array[i] > max) {
+        max = array[i];
+      }
+    }
+    return max;
+  }
+
   public static void radixSort(int[] ar) {
-    // TODO: https://github.com/williamfiset/Algorithms/issues/5
+    int bucket[][] = new int[10][10];
+    int bucket_count[] = new int[10];
+    int i, j, k, remainder, numDig = 0, divisor = 1, largest, passNo;
```

Comment:
Please reduce the scope of all these variables as much as possible to make the code easier to follow? This is how bugs happen.